adoc/sap-nw740-sle15-setupguide.adoc

```diff
@@ -752,7 +752,8 @@ For the ERS and ASCS instance edit the instance profiles
 profile directory _/usr/sap/{mySid}/SYS/profile/_.
 
 You need to tell `{sapStartSrv}` to load the HA script connector library and
-to use the `{s4sClConnector3}`.
+to use the `{s4sClConnector3}`. On the other hand, please make sure the feature
+_Autostart_ is *not* used.
```

Comment:
```
On the other hand, please make sure the feature
_Autostart_ is *not* used.
```
should be changed to
Additionally, make sure the feature _Autostart_ is *not* used.